docs/docs/schema-customization.md

```diff
@@ -89,6 +91,7 @@ type Internal {
   type: String!
 }
 ```
+Here above its the basic schema created using Graphql. As you can see some ! marks.Here String! or ID! ... means that the field is non-nullable, meaning that the GraphQL service promises always to give you a value when you query this field. In the type language, we'll represent those with an exclamation mark (!). If in case it is [Node!] then it means all the elements of the array of values containing Node values are also Non-nullable. So as a whole children always contain values and non-nullable.
```

Comment:
This seems fairly repetitive (particularly the use of means/meaning/mean) and seems to explain in a bit of a circle.